### PR TITLE
ab2d 667 shutdown cleanup reset jobs

### PR DESCRIPTION
**Pull request by spathiyil**
When a shutdown command is issued, worker needs to put active jobs into submitted status

### Summary

When a shutdown command is issued, worker needs to put active jobs into submitted status so that the job can be processed by another instance.

### Checklist

- [x] Tests and linting pass

### Security
N/A

### Additional JIRA Tickets (optional)

N/A

### CODE REVIEW

File: worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java [hunk 1]
```diff
@@ -16,17 +21,37 @@
 
     private final JobPreProcessor jobPreprocessor;
     private final JobProcessor jobProcessor;
+    private final ShutDownService shutDownService;
+
+    private static List<String> activeJobs = Collections.synchronizedList(new ArrayList<>());
```
Comment (dkrylovsb) [inline worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java]: Does it need to be `static`? I _think_ the Worker bean is a singleton, but even if it is not, each instance would maintain the jobs it is processing anyway.

File: worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java [hunk 2]
```diff
@@ -16,17 +21,37 @@
...
+
+        activeJobs.add(jobUuid);
+
         jobPreprocessor.preprocess(jobUuid);
         log.info("Job was put in progress");
 
         jobProcessor.process(jobUuid);
         log.info("Job was processed");
+
+        activeJobs.remove(jobUuid);
```
Comment (dkrylovsb) [inline worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java]: We need a `try/finally` block here and remove the `jobUuid` from the list in the `finally` clause.

File: worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java (hunk 1 above)
Comment (spathiyil) [reply worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java]: done

File: worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java (hunk 2 above)
Comment (spathiyil) [reply worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java]: done

**Pull request merged**